lib/ansible/modules/cloud/amazon/elb_classic_lb.py

```diff
@@ -1063,7 +1063,10 @@ def select_stickiness_policy(self):
                     if 'expiration' not in self.stickiness:
                         self.module.fail_json(msg='expiration must be set when type is loadbalancer')
 
-                    expiration = self.stickiness['expiration'] if self.stickiness['expiration'] is not 0 else None
+                    try:
+                        expiration = self.stickiness['expiration'] if int(self.stickiness['expiration']) else None
```

Comment:
ie: if self.stickiness['expiration'] is the string "10", the change being made does the comparison of self.stickiness['expiration'] as the int 10, but the variable expiration is set to the string "10".